src/services/translate.ts

```diff
@@ -0,0 +1,12 @@
+import Languages from '../lang';
+import { store } from '../index';
+
+const DEFAULT_LANGUAGE_KEY = 'en';
+
+export default (translationKey: string) => {
+    const languageKey = store.getState().user.language;
```

Comment:
@ichsaras hoe zwaar is deze specifieke call?
Loont het meer om alle componenten als 2e argument een languagekey mee te laten geven aan deze functie, en een connect op alle componenten te zetten die een languagekey willen hebben?
Dat word nogal een refactor, maar als het veel uitmaakt is het het wel waard imo.